src/asmjs/asm_v_wasm.cpp

```diff
@@ -75,12 +90,18 @@ std::string getSig(Function *func) {
 
 Type sigToType(char sig) {
   switch (sig) {
-    case 'i': return i32;
-    case 'j': return i64;
-    case 'f': return f32;
-    case 'd': return f64;
-    case 'v': return none;
-    default: abort();
+    case 'i':
```

Comment:
I wonder if we can find a setting to allow this kind of thing?

Reply:
Setting `AllowShortCaseLabelsOnASingleLine` to true allows this, but settings also means converting all two-line case statements into a single line if they fit in 120 cols, the current `ColumnLimit`. There are several options like this; refer to the bottom of https://github.com/WebAssembly/binaryen/pull/1407#issuecomment-365224159 for details.

So we have to either convert all
```cpp
case 'i':
  blahblah;
```
into 
```cpp
case 'i': blahblah;
```

or we have to convert all
```cpp
case 'i': blahblah;
```
into
```cpp
case 'i':
  blahblah;
```

So the point is, even though the option name is `AllowShortCaseLabelsOnASingleLine`, there is no option that _allows_ something. clang-format, in nature, forces a single style for given code and configurations. The option should read as `ForceShortCaseLabelsOnASingleLine`.

Merging all two-line case statements (or if statements, ...) into a single line may not sound that bad, but we also have `ColumnLimit` of 120, which is longer than usual. This will merge a lot of two-line case statements into one. So, I was not able to decide, so I ended up not adding those `Allow***` options. We can add them if people prefer that.